environments/kolla/configuration.yml

```diff
@@ -46,6 +46,7 @@ elasticsearch_curator_hard_retention_period_days: 1
 
 # keystone
 enable_keystone_federation: "yes"
+keystone_oidc_forward_header: "X-Forwarded-Proto"
```

Comment:
@JuanPTM  Perhaps we should comment it  with #  so for upgrade pathes it won't block and for deployments with newer openstack your able to switch on